src/code_805D8C8_1.c

```diff
@@ -0,0 +1,4835 @@
+extern void sub_8045DB4(Position *, u32);
+extern s32 sub_8052B8C(u32, const u8 *, u32);
+bool8 sub_807EF48(void);
+void sub_806A2BC(Entity *a0, u8 a1);
+bool8 sub_805E874(void);
+bool8 sub_80701A4(Entity *a0);
+void sub_80647F0(Entity *a0);
+void sub_805E738(Entity *a0);
+void sub_803E708(s32 a0, s32 a1);
+void sub_8040A78(void);
+void sub_805E804(void);
+void sub_8049ED4(void);
+void sub_8064BE0(void);
+void sub_8075680(u32);
+void sub_8094C88(void);
+void sub_8040A84(void);
+void sub_8047158(void);
+void sub_804AA60(void);
+void sub_80532B4(void);
+void sub_806A914(u8 a0, u8 a1, u8 a2);
+void sub_8044C10(u8 a0);
+u16 GetLeaderActionId(void);
+void sub_80978C8(s16 a0);
+void sub_8044C50(u16 a0);
+void sub_805E2C4(Entity *leader);
+bool8 sub_8094C48(void);
+void sub_8052210(u8 a0);
+bool8 sub_805EC4C(Entity *a0, u8 a1);
+void sub_803E724(s32 a0);
+void HandleTalkFieldAction(Entity *);
+bool8 sub_8044B28(void);
+bool8 IsNotAttacking(Entity *param_1, bool8 param_2);
+void ShowFieldMenu(u8 a0, bool8 a1);
+bool8 sub_805EF60(Entity *a0, EntityInfo *a1);
+s32 GetTeamMemberEntityIndex(Entity *pokemon);
+bool8 sub_8070F80(Entity * pokemon, s32 direction);
+void DrawFieldMenu(u8 a0);
+bool8 sub_805FD74(Entity * a0, struct UnkMenuBitsStruct *a1);
+bool8 sub_805FD3C(struct UnkMenuBitsStruct *a0);
+void sub_8060D24(UNUSED ActionContainer *a0);
+bool8 sub_8060E38(Entity *a0);
+void sub_8062D8C(ActionContainer *a0);
+void sub_80637E8(ActionContainer *a0);
+void sub_8063B54(ActionContainer *a0);
+void sub_8063BB4(ActionContainer *a0);
+void sub_806752C(ActionContainer *a0);
+bool8 sub_8061A38(ActionContainer *a0, bool8 a1);
+void sub_8063A70(ActionContainer *a0, bool8 a1);
+void sub_8063CF0(ActionContainer *a0, bool8 a1);
+void sub_8067768(UNUSED ActionContainer *a0);
+void ShowTacticsMenu(ActionContainer *a0);
+void sub_804A728(Position *pos, s32 a1, u8 a2, u8 a3);
+extern bool8 sub_8071A8C(Entity *pokemon);
+extern void sub_80643AC(Entity *pokemon);
+extern u8 sub_8062F90(Entity *, u32, u32, u32, u32);
+
+extern Entity *gLeaderPointer;
+
+extern u8 gUnknown_202F22D;
+extern u8 gUnknown_202F22C;
+extern u8 gUnknown_202F230;
+extern u8 gUnknown_202F231;
+extern u8 gUnknown_202EE00;
+extern u16 gUnknown_202F22E;
+
+extern const u8 *gUnknown_80F8A84;
+extern const u8 *gUnknown_80F8A6C;
+extern const u8 *gUnknown_80F8ADC;
+extern const u8 *gUnknown_80F8AB0;
+extern const u8 *gUnknown_80F8B0C;
+extern const u8 *gUnknown_80FD4B0;
+extern const u8 *gUnknown_80F8A4C;
+extern const u8 *gUnknown_80F8A28;
+extern const u8 *gUnknown_8100208;
+extern const u8 *gUnknown_80F9BD8;
+extern const u8 *gUnknown_80F9C08;
+extern const u8 *gUnknown_80F9C2C;
+extern const u8 *gUnknown_80F9BB0;
+extern const u8 *gUnknown_80FDE18;
+extern const u8 *gUnknown_80F8B24;
+extern const u8 *gTeamToolboxAPtr;
+extern const u8 *gTeamToolboxBPtr;
+extern const u8 *gFieldItemMenuGroundTextPtr;
+extern const u8 *gUnknown_80FE940;
+extern const u8 *gWhichTextPtr1;
+
+#ifdef NONMATCHING
+
```

Comment:
If you're ready to merge this in, add the decomp.me link as a comment here so others can easily pick it up.